### PR TITLE
add dill package

### PR DESCRIPTION
**Pull request by yguezpa**
## Status
Ready/In Progress/In Hold Ready

## Related Issues
related: 26297

## Description
Added a new package to the existing docker sklearn

### CODE REVIEW

File: docker/sklearn/Pipfile [hunk 1]
```diff
@@ -10,6 +10,8 @@ numpy = "*"
 scikit-learn = "*"
 pandas = "*"
 nltk = "*"
+dill = "*"
+eli5 = "==0.10.1"
```
Comment (glicht) [inline docker/sklearn/Pipfile:14]: Why specific version? Why not *?

**Comment by yguezpa** [comment]
> You added Pipfile and Pipfile.lock in the root dir that need to be removed

@glicht I need to remove Pipfile and Pipfile.lock and after install eli5 package?

File: Pipfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[[source]]
```
Comment (glicht) [inline Pipfile:1]: This file seems to have been checked in by mistake

File: Pipfile.lock [hunk 3]
```diff
@@ -0,0 +1,196 @@
+{
```
Comment (glicht) [inline Pipfile.lock:1]: this file seems to have been checked in by mistake

File: Pipfile [hunk 4]
```diff
@@ -6,10 +6,8 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-numpy = "*"
-scikit-learn = "*"
-pandas = "*"
-nltk = "*"
+eli5 = "==0.10.1"
+dill = "==0.3.2"
```
Comment (glicht) [inline Pipfile:10]: Why do you have an exact version and not star? I would recommend changing to * so the package will be auto updated by depandabot.

**Comment by glicht** [comment]
Was handled in a different pr
